lib/auxiliary_data_computation.rb

```diff
@@ -13,10 +13,16 @@ def self.compute_concise_results
       %w(average ConciseAverageResults),
     ].each do |field, table_name|
       ActiveRecord::Base.transaction do
```

Comment:
There's probably no need for a transaction anymore, because DDL statements like `CREATE TABLE` implicitly commit transactions anyway (this varies across DBMS, but I understand [this is the case](https://dev.mysql.com/doc/refman/8.4/en/implicit-commit.html) for MySQL). It should be fine, since we swap the tables only if the new one was successfully computed.

Reply:
Replaced the `transaction` wrapper with a `begin`/`ensure` wrapper